oss-internship-2020/curl/examples/example1.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sandboxed version of simple.c
+// Simple HTTP GET request
+
+#include <cstdlib>
+
+#include "../sandbox.h"
+
+int main(int argc, char* argv[]) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  google::InitGoogleLogging(argv[0]);
+
+  absl::Status status;
+  sapi::StatusOr<CURL*> status_or_curl;
+  sapi::StatusOr<int> status_or_int;
+
+  // Initialize sandbox2 and sapi
+  CurlSapiSandbox sandbox;
+  status = sandbox.Init();
+  if (!status.ok()) {
+    LOG(FATAL) << "Couldn't initialize Sandboxed API: " << status;
+  }
+  CurlApi api(&sandbox);
+
+  // Initialize the curl session
+  status_or_curl = api.curl_easy_init();
```

Comment:
Name according to the contained value.
So in this case e.g. `curl_handle`
Declare variables close to where they are used.